src/publish_control_board_rev2.cpp

```diff
@@ -282,3 +282,7 @@ void PublishControlBoardRev2::publish_lights_horn_wipers_message(const sensor_ms
     }
   }
 }
+
+void PublishControlBoardRev2::publish_door_signal_message(const sensor_msgs::Joy::ConstPtr& msg)
```

Comment:
Why is this function empty? Does rev2 of the board not support door signals?

Reply:
Correct! Actually I just need that function in PublishControlBoardRev3. But I had to define it in PublishControlBoardRev2 as well, I guess because it's declared virtual in publish_control.h